zuul.d/nodeset.yaml

```diff
@@ -53,17 +23,6 @@
       - name: peers
         nodes: []
 
-- nodeset:
-    name: multinode-centos-9-stream-crc
```

Comment:
does this need a rebase? i cannot find it in https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/nodeset.yaml

Reply:
yes seems removed with https://github.com/openstack-k8s-operators/ci-framework/commit/87e215703394f711e3f4cd95ebfd3e1000740d8e